org.simondalvai.ball2box.yml

```diff
@@ -0,0 +1,39 @@
+app-id: org.simondalvai.ball2box
+runtime: org.freedesktop.Platform
+runtime-version: 22.08
+sdk: org.freedesktop.Sdk
+base: org.godotengine.godot.BaseApp
+base-version: 3.5
+command: godot-runner
+finish-args:
+  - --share=ipc
+  - --socket=x11 # <-- Godot needs X11
+  - --socket=pulseaudio
+modules:
+  - name: Ball2Box
+    buildsystem: simple
+    build-commands:
+      # The game itself
+      - tar -xvf ball2box.tar.gz
+      - install Ball2Box.pck /app/bin/godot-runner.pck
+      # Metadata and a .desktop file
+      - install -D org.simondalvai.ball2box.desktop /app/share/applications/org.simondalvai.ball2box.desktop
+      - install -D org.simondalvai.ball2box.appdata.xml /app/share/metainfo/org.simondalvai.ball2box.appdata.xml
+      # Icons
+      - install -D icon-512.png /app/share/icons/hicolor/512x512/apps/org.simondalvai.ball2box.png
+      - install -D icon-256.png /app/share/icons/hicolor/256x256/apps/org.simondalvai.ball2box.png
+      - install -D icon-128.png /app/share/icons/hicolor/128x128/apps/org.simondalvai.ball2box.png
+      - install -D icon-64.png /app/share/icons/hicolor/64x64/apps/org.simondalvai.ball2box.png
+    sources:
+      # The game is pulled from GitHub
+      - type: file
+        url: https://github.com/dulvui/ball2box/releases/download/v4.1.0/ball2box.tar.gz
+        archive-type: tar-gz
```

Comment:
as the warnings point out this is an invalid property. but really you should use type: archive and let flatpak handle it.

Reply:
I use archive in tar.xz format now, but how can I access the content of it in build-commands? I always get file not found